templates/nats.conf.j2

```diff
@@ -12,6 +12,14 @@ log_file: "{{ nats_log_dir }}/{{ nats_log_file }}"
 debug: {{ nats_log_debug|lower }}
 {% endif %}
 
+{% if nats_jetstream_enabled|bool %}
+jetstream {
```

Comment:
That's OK for me not to have per account settings and only global ones (i.e. we aren't going to restrict account's jetstream resources) but could you please document all new option and mention this option has "global" effect?